OWNERS

```diff
@@ -5,11 +5,20 @@ approvers:
 - elsony
 - kim-tsao
 - maysunfaisal
-- mik-dass
 - yangcao77
+- johnmcollier
+- michael-valdron
+- thepetk
+- feloy
+- rm3l
```

Comment:
do we need all these new folks on the reviewers and approvers list?

Reply:
updated!